templates/flipcard.jsx

```diff
@@ -19,11 +19,12 @@ export default function flipcard(props) {
               `component__item flipcard__item item-${index} ${_flipDirection}`,
             ])}
             key={index}
+            tabIndex='0'
```

Comment:
Is this one needed? I have a feeling it may disrupt the a11y flow of the component

Reply:
It is needed to allow the item to take the focus-visible class, however with or without it I am seeing the same behaviour / a11y flow using keyboard and enabling voiceover.